NuRadioMC/simulation/config_default.yaml

```diff
@@ -13,6 +13,9 @@ speedup:
   redo_raytracing: False  # redo ray tracing even if previous calculated ray tracing solutions are present
   min_efield_amplitude: 2  # the minimum signal amplitude of the efield as a factor of the noise RMS. If the value is smaller, no detector simulation is performed. As the vector effecive length of antennas is typically less than 1, this cut does not introduce any bias as long as the value is smaller than the trigger threshold.
   amp_per_ray_solution: True  # if False, the maximum aplitude for each ray tracing solution is not calculated
+  distance_cut: True # if True, a cut for the vertex-observer distance as a function of shower energy is applied (log10(max_dist) = intercept + slope * log10(shower_energy))
```

Comment:
better make the default False. And can you add units to the equation? I think it should be log10(max_dist / m) = intercept + slope * log10(shower_energy / eV)

Reply:
That's right, I've forgotten.

Reply:
Done.